adafruit_bme680.py

```diff
@@ -33,6 +33,13 @@
 import math
 from micropython import const
 
+try:
+    # Used only for type annotations.
```

Comment:
Since we only want this to run in a computer (development) environment, but skip on the microcontroller, we can use the `typing` module to guard from running the rest of the block (since CircuitPython doesn't implement the `typing` module):

```suggestion
    # Used only for type annotations.
    import typing  # pylint: disable=unused-import
```

The `pylint` disable will allow us to still try importing the module while suppressing the typical pylint error :)